### PR TITLE
LPAL-1174 Restructure renovate config

### PR DESCRIPTION
**Pull request by townxelliot**
## Purpose

[LPAL-1174](https://opgtransform.atlassian.net/browse/LPAL-1174)

## Approach

* Update cypress Dockerfile and package*.json files at the same time.
* Don't do any major updates.
* Prevent updates to php, python, redis and postgres.
* Don't update PHP PDF libraries.

## Learning

Renovate is not my friend.

## Checklist

* [X] I have performed a self-review of my own code
* [X] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [X] I have added tests to prove my work
* [X] I have added mandatory tags to terraformed resources, where possible
* [X] If I have a new OPG component dependency, I have updated the `metadata.json` with the repo location.
* [X] If I added a package.json or composer.json, I also made sure this is included in the script in `.github/workflows/dependabot-update.yml`
* [ ] The product team have tested these changes

[LPAL-1174]: https://opgtransform.atlassian.net/browse/LPAL-1174?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -27,112 +19,62 @@
...
-      "matchDatasources": [
-        "docker"
-      ],
-      "matchPackageNames": [
-        "postgres",
-        "php"
+        "No updates to PHP PDF libraries (too sensitive and risky)"
       ],
+      "matchUpdateTypes": ["major", "minor", "patch"],
+      "matchManagers": ["composer"],
```
Comment (mabely) [inline renovate.json:40]: Is `matchDatasources` different and how?

File: renovate.json [hunk 2]
```diff
@@ -27,112 +19,62 @@
...
-  "major": {
-    "labels": [
-        "Dependencies",
-        "Renovate"
-    ],
-    "prCreation": "not-pending",
-    "prPriority": 0
-  },
-  "prHourlyLimit": 1,
+  "prHourlyLimit": 10,
```
Comment (mabely) [inline renovate.json:25]: Would it be preferable for this to be lower lest we be flooded by renovate PRs again?

File: renovate.json (hunk 1 above)
Comment (townxelliot) [reply renovate.json:53]: I found it easier to use managers, as it's difficult to work out what the datasources are. I honestly don't know the difference.

File: renovate.json (hunk 2 above)
Comment (townxelliot) [reply renovate.json]: I have lowered it.

**Review by nickdavis2001 — APPROVED** [review]

**Pull request merged**